provd/internal/services/user/password_hash.go

```diff
@@ -2,7 +2,7 @@
 package user
 
 /*
-#cgo CFLAGS: -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include
+#cgo pkg-config: glib-2.0 gio-2.0 gobject-2.0
```

Comment:
do you need to include the 3 of them? Also, the "ci.yml" still include a hardcoded lib path then?